test/test_fake_tensor.py

```diff
@@ -0,0 +1,80 @@
+# Owner(s): ["module: unknown"]
```

Comment:
well, could always chuck it in module: meta tensors

Reply:
`test_meta` is under `module: primTorch`.. I dont think there is a meta module